jekyll/_cci2/install-machine-runner-3-on-docker.adoc

```diff
@@ -0,0 +1,117 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.4+
+---
+= Install machine runner 3.0 on Docker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions on how to install CircleCI's self-hosted machine runner (3.0) on Docker.
+:icons: font
+:experimental:
+:machine:
+:docker:
+
+This page describes how to install CircleCI's machine runner 3.0 with the Docker executor. If you are looking to set up self-hosted runners in a private Kubernetes cluster, visit the <<container-runner-installation#,Container runner>> page.
+
+[NOTE]
+====
+xref:container-runner#[Container runner] is the **recommended method** for self-hosted runner Docker installation. The instructions on this page are for a simple Docker setup using machine runner 3.0.
```

Comment:
We had some confusion in a call with horizon yesterday when I was trying to explain that docker runners are machine runners. Could a CI config example like we have for machine runner 3 (https://circleci.com/docs/install-machine-runner-3-on-linux/#machine-runner-configuration-example) be added in here as well so we avoid folks trying to use the docker key?

Reply:
That is already included in the web app [install snippet](https://internal.circleci.com/docs/runner-installation-docker/#create-namespace-and-resource-class)

I would like to think that plus all the references to it being a machine runner is sufficient?

Reply:
Ignore that 😅 I have added the snippet used on the page you linked